### ISSUE TITLE
Interactive timeseries graph 

### ISSUE THREAD

**Issue by JIMENOFONSECA** [architecture-building-systems/CEAforArcGIS#318]
we discussed about this in the past. It would be great to use .3ds or any other library in java to produce more interactive graphs for the tool.

**Comment by daren-thomas** [comment]
How do I test this?

**Comment by daren-thomas** [comment]
OK, I figured it out: Just run the script!

@JIMENOFONSECA: Should this really be in this milestone? Adding functionality like this includes at least two more issues to implement:
1. Update of the documentation
2. Adding to toolbox in ArcGIS (incl. testing)

**Comment by daren-thomas** [comment]
It runs, but is super slow...

**Comment by JIMENOFONSECA** [comment]
I believe we can keep it in as an script working for research purposes. So it does not need documentation nor adding it to the toolbox. It can be just like the optimisation part we added before and that does not have an interface. Would this work?


On 25 Aug 2016, at 5:18 pm, Daren Thomas <notifications@github.com<mailto:notifications@github.com>> wrote:


OK, I figured it out: Just run the script!

@JIMENOFONSECA<https://github.com/JIMENOFONSECA>: Should this really be in this milestone? Adding functionality like this includes at least two more issues to implement:
1. Update of the documentation
2. Adding to toolbox in ArcGIS (incl. testing)
